### PR TITLE
HAWQ-805. Add feature test for exttab1 with new framework

### PR DESCRIPTION
**Pull request by xunzhang**

### CODE REVIEW

**Comment by xunzhang** [comment]
cc @huor @yaoj2

File: src/test/feature/ExternalSource/sql/exttab1.sql.source [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+-- 0|AFRICA|lar deposits. blithely final packages cajole. regular waters are final requests. regular accounts are according to 
+-- 1|AMERICA|hs use ironic, even requests. s
+-- \.
+INSERT INTO wet_region SELECT * from reg_region;
+
+-- COPY wet_region FROM STDIN DELIMITER '|';
+-- 2|ASIA|ges. thinly even pinto beans ca
+-- 3|EUROPE|ly final courts cajole furiously final excuse
+-- \.
+-- INSERT INTO wet_region VALUES(4,'MIDDLE EAST','uickly special');
```
Comment (xunzhang) [inline src/test/feature/ExternalSource/sql/exttab1.sql.source]: Temporary comment, for future debug, it is init fail which does not relate to this pull request.

File: src/test/feature/ExternalSource/sql/exttab1.sql.source [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+
+--
+-- Now use RET to see if data was exported correctly.
+-- NOTE: since we don't bother cleaning up the exported file, it may grow bigger
+-- in between runs, so we don't check for count(*), and instead check for distinct.
+--
+-- SELECT DISTINCT(R_NAME) FROM ret_region ORDER BY R_NAME;
```
Comment (xunzhang) [inline src/test/feature/ExternalSource/sql/exttab1.sql.source]: Temporary comment.

**Comment by huor** [comment]
+1

File: src/test/feature/ExternalSource/sql/exttab1.sql.source (hunk 1 above)
Comment (xunzhang) [inline src/test/feature/ExternalSource/sql/exttab1.sql.source]: The problem is resolved: https://github.com/apache/incubator-hawq/commit/0bc49a345bb97d3d64980164b0fa9e4601c5953d

**Comment by yaoj2** [comment]
LGTM

**Comment by xunzhang** [comment]
Merged into master, thanks.

**Pull request closed without merging**